src/App.scss

```diff
@@ -1,39 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
 iframe {
   display: none;
 }
 
 html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
 }
 
-label,
-button {
-  cursor: pointer;
+body,
+h1,
+p {
+  margin: 0;
+}
+
+body {
+  background-color: #333;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 50px 20px;
+  max-width: 1024px;
+
+  &__block {
+    margin-bottom: 20px;
+    padding: 20px;
+    background-color: #534f4f;
+    border-radius: 5px;
+  }
+}
+
+.title {
+  margin-bottom: 40px;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
 }
 
 .error {
+  padding-left: 10px;
+  font-size: 16px;
   color: red;
 }
 
-.TodoInfo {
-  width: max-content;
-  padding: 8px;
-  margin: 12px 0;
-  border: 1px solid #000;
-  border-radius: 8px;
-  background-color: antiquewhite;
+.form {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 
-  &__title {
-    margin: 4px 0;
-    font-size: inherit;
-    color: #f00;
+  &__fields {
+    display: flex;
   }
 
-  &--completed &__title {
-    color: #080;
+  &__field {
+    margin-right: 20px;
+  }
+
+  &__label {
+    margin-bottom: 8px;
+    font-size: 20px;
+    color: #fff;
+  }
+
+  &__input {
```

Comment:
Consider using a more descriptive class name instead of `input`. For example, `__form-input` or `__text-field`. 🤔